Packs/Cymptom/Integrations/Cymptom/Cymptom.py

```diff
@@ -0,0 +1,250 @@
+
+    privileged = argToBoolean(args.get("privileged", "True"))
+
+    for mitigation in mitigations_results["mitigations"]:
+
+        if mitigation["name"] == "Brute Force":
+            mitigation_id = mitigation["id"]
+            break
+
+    if mitigation_id:
+        l_mitigations = client.get_mitigation_by_id(mitigation_id)
+
+        for subtechnique in l_mitigations["subtechniques"]:
+            if subtechnique["name"] == "Password Guessing":
+                procedures_ids.add(subtechnique["id"])
+
+    if procedures_ids:
+
+        for proc_id in procedures_ids:
+            l_users = client.get_mitigations_subtechnique_procedure_by_id(proc_id)
+
+            if l_users:
+                l_users = l_users[0]["targets"]
+                for user in l_users:
+                    if "Domain" in user["labels"]:
+                        username_dict = user["name"]
+                        if privileged and ("DomainAdmin" in user["labels"] or "ComputerAdmin" in user["labels"]):
+                            privileged_users.append(username_dict)
+                        else:
+                            unprivileged_users.append(username_dict)
+
+    if privileged_users:
+        for username in privileged_users:
+            users_formatted.append({"Username": username})
+        readable_output = tableToMarkdown('Privileged Users With Cracked Passwords', privileged_users,
+                                          headers=table_headers)
+    elif unprivileged_users:
+        for username in unprivileged_users:
+            users_formatted.append({"Username": username})
+        readable_output = tableToMarkdown('Unprivileged Users With Cracked Passwords', unprivileged_users,
+                                          headers=table_headers)
+    else:
+        readable_output = tableToMarkdown('Users With Cracked Passwords', unprivileged_users,
+                                          headers=table_headers)
+
+    command_results = CommandResults(
+        outputs_prefix="Cymptom.CrackedUsers",
+        outputs_key_field="Username",
+        readable_output=readable_output,
+        outputs=users_formatted,
+    )
+
+    return command_results
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    LOG(f'Command being called is: {demisto.command()}')
+
+    params = demisto.params()
+
+    base_url = params['url']
+
+    api_key = params['api_key']
+
+    # Flag if use server proxy
+    use_proxy = params.get('proxy', False)
+
+    # Flag if use server 'verification'
+    insecure = params.get('insecure', False)
+
+    headers = {
+        "Authorization": f"Bearer {api_key}"  # Replace ${token} with the token you have obtained
+    }
+
+    demisto.debug(" ---- MAIN CALL -----")
+    demisto.debug(" ---- PARAMS -----")
+    demisto.debug(f"base_url: {base_url}")
+    demisto.debug(f"api_key: {api_key}")
+    demisto.debug(f"insecure: {insecure}")
+    demisto.debug(f"use_proxy: {use_proxy}")
```

Comment:
Also, let's remove these. You can find all the integration's parameters in the log bundle.
If you still like to keep those, please remove the api_key log. it is not secured and this field is encrypted.

Reply:
Isn't the api key controlled only by the user aa the user is the one who filled it?
Also, where can I find the log bundle?

Reply:
Isn't the api key controlled only by the user aa the user is the one who filled it?
Also, where can I find the log bundle?

Reply:
I dont fully understand what you mean by saying that the API key is controlled by the user.
API key is not something that we want to log. it creates a security hole.

You can read about log bundle [here](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/logs/create-a-log-bundle.html)
and read about executing a command with `debug-mode=true` [here](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/logs/create-a-log-bundle.html)

Reply:
Gotcha.